main.go

```diff
@@ -173,6 +173,7 @@ func main() {
 		disp = NewDispatcher(alerts, NewRoute(conf.Route, nil), build(conf.Receivers), marker)
 
 		go disp.Run()
+		log.Infof("Alertmanager is now listening on port %s", strings.TrimPrefix(*listenAddress, ":"))
```

Comment:
Hello,

I think it would be more appropriate to put this log above `go listen(router)`.
Also, why not just display the full `listenAddress` like it is done in [`prometheus/prometheus`](https://github.com/prometheus/prometheus/blob/master/web/web.go#L239) ?
```
log.Infof("Listening on %s", *listenAddress)
```